typings/primordials.d.ts

```diff
@@ -480,6 +480,8 @@ declare namespace primordials {
   export const SymbolFor: typeof Symbol.for
   export const SymbolKeyFor: typeof Symbol.keyFor
   export const SymbolAsyncIterator: typeof Symbol.asyncIterator
+  export const SymbolDispose: typeof Symbol
+  export const SymbolAsyncDispose: typeof Symbol
```

Comment:
Can you add a todo in here to fix these when these are available?

Reply:
Done